tests/vxlan/vnet_config.py

```diff
@@ -0,0 +1,11 @@
+IPV6_VXLAN_TEST = False
```

Comment:
`vnet_params` might be a more appropriate name. There's also an argument to be made for putting this in `__init__.py`. What do you think?

Reply:
The reason I named it `vnet_config` is because the canonical way to share vars like this is placing them in a file name `config` or `cfg` (added `vnet` to prevent possible name conflicts)([source](https://docs.python.org/3/faq/programming.html#how-do-i-share-global-variables-across-modules)). Would be open to changing the name, just wanted to make this known.